internal/characters/furina/burst.go

```diff
@@ -37,7 +36,7 @@ func (c *char) addFanfare(amt float64) {
 	if c.Base.Cons >= 2 {
 		amt *= 3.5
 	}
-	c.curFanfare = common.Min(c.maxFanfare, c.curFanfare+amt)
+	c.curFanfare = math.Min(c.maxFanfare, c.curFanfare+amt)
```

Comment:
You were supposed to use the builtin min/max, not math.Min/Max. Please adjust that for Furina in all files.

Reply:
Can I do it for all the Golang files or just Furina?

Reply:
Just Furina pls.